transcom/mymove
# INT B-18751 History Log PPM Shipment Created

## [B-18751](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-18751)

## Summary

Quick, easy change. Uses the existing `createMTOShipment.jsx` file, and changes the title if the shipment is a PPM. All other requested info was already recorded. I also removed the "STATUS: DRAFT" portion, since this was always true upon shipment creation.

## How to Test

Create a move with a PPM shipment, then sign and send to SC. Log in as SC (or any role), and find your move code using the search tab.

In the Move History tab, near the end of the first page or maybe on the second page, you should see an entry like this:
![Screenshot 2024-03-26 at 2 38 53 PM](https://github.com/transcom/mymove/assets/146959895/d70fc185-bbfa-49f4-ba4a-a7b405faa65a)
